Tests/test_dict.py

```diff
@@ -472,23 +472,13 @@ def test_dict_to_idict(self):
         self.load_iron_python_test()
         from IronPythonTest import DictConversion
         class MyDict(dict): pass
-        class KOld: pass
-        class KNew(object): pass
-        class KOldDerived(KOld): pass
-        class KNewDerived(KNew): pass
 
         test_dicts = [
                         {},
                         {1:100},
-                        {None:None},
                         {object:object},
                         {1:100, 2:200},
                         {1:100, 2:200, 3:300, 4:400},
-                        MyDict.__dict__,
```

Comment:
It's too bad we're losing the check since `MyDict.__dict__` is actually `mappingproxy` and not a `dict` like the remaining values. Maybe we could change `DictConversion.ToIDictionary` to return tuples instead? Then we'd be able to sort and compare.